1bash.template

```diff
@@ -608,8 +610,9 @@ plusCPU_WALLET_FORMAT="$WALLET_ADDRESS_FORMAT"
 ########################################################
 
 CLAYMORE_VERSION="latest"                        # Choose: latest or any version you add its folder.
-ETHMINER_OPTS=""                                 # Ethminer optional arguments. add "--api-port -3333" if you want web info
 CLAYMORE_OPTS=""                                 # Claymore optional arguments. EX: add "-esm 2" for miningpoolhub
+ETHMINER_OPTS=""                                 # Ethminer optional arguments. add "--api-port -3333" if you want web info
+ETHMINER_SCHEMA="stratum+tcp"                    # Use Ethminer new schema; stratum+ssl, stratum+tcp, stratum+tls, stratum+tls12, stratum1+ssl, stratum1+tcp, stratum1+tls, stratum1+tls12, stratum2+ssl, stratum2+tcp, stratum2+tls, stratum2+tls12
```

Comment:
Ok, just drop this line and move the comment  in the notes of the coin section and this is ready.